content/guides/debugging/_index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Incident Debugging Workflow"
+date: 2018-11-26T22:22:31-07:00
```

Comment:
Please update this date to today's date.

Reply:
Done, please see new commit.